sheepdog/config.py

```diff
@@ -21,3 +21,5 @@
     logger.info(
         "Running in ACTIVE mode. The Elasticsearch 'GDC_ES_INDEX' "
         "environment variable and the active mapping will be used. ")
+
+REQUIRE_FILE_INDEX_EXISTS = False
```

Comment:
This shouldn't be here, it should be in https://github.com/uc-cdis/sheepdog/blob/master/sheepdog/test_settings.py instead